client/app/dbaas/cloud-db/advanced-parameter/cloud-db-advanced-parameter-edit.controller.js

```diff
@@ -0,0 +1,57 @@
+class CloudDbAdvancedParameterEditCtrl {
+    constructor ($state, $stateParams, CloudDbAdvancedParameterService, CloudDbProjectService, CloudNavigation, ControllerHelper) {
+        this.$state = $state;
+        this.$stateParams = $stateParams;
+        this.CloudDbAdvancedParameterService = CloudDbAdvancedParameterService;
+        this.CloudNavigation = CloudNavigation;
+        this.ControllerHelper = ControllerHelper;
+
+        this.projectId = $stateParams.projectId;
+        this.instanceId = $stateParams.instanceId;
+
+        this.initialValues = {};
+        this.model = {};
+
+        this.parameters = this.ControllerHelper.request.getHashLoader({
```

Comment:
You don't have errorHandler I don't know if it is the normal way or a forgotten things.